script/Arcueid/Day 9/09_00_ARC09_2_2.txt

```diff
@@ -191,17 +192,17 @@ But the only thing that left an impression was being called to the old man's stu
 [sha:d2cd41810eebb2bdaafdbfbbdbd961ed1985479d]{
 -- Page 88, Offset 21530.
 -- その時、おまえは遠野家の長男に相応しくない、と絶縁状を突きつけられたんだっけ。
-At that time, I was thrusted down with a letter of disownment, saying that I was not worthy to be the eldest son of the Tohno family.
+At that time, I was thrusted with a letter of disownment, saying that I was not worthy to be the eldest son of the Tohno family.
```

Comment:
At that time, a letter of disownment was thrust before me saying that I was not worthy to be the eldest son of hte Tohno family.